index.html

```diff
@@ -12,140 +12,158 @@
-      <p class="my-button"><span> Get my CV <i class="fa fa-download" aria-hidden="true"></i></span> </p>
-
-      <div class="about-info bg-blue">
-        <p class="about-title">Languages</p>
-        <ul>
-          <li>JavaScript (ES5/ES6)</li>
-            <li>Ruby</li>
-            <li>HTML</li>
-            <li>CSS</li>
-        </ul>
-      </div>
-      <div class="about-info bg-red">
-        <p class="about-title">Frameworks</p>
-        <ul>
-          <li>Bootstrap</li>
-            <li>Ruby on rails</li>
-            <li>RSpec</li>
-            <li>CSS</li>
-        </ul>
+    <section class="section-title about vw-10-margin">
+      <div class="about-intro display-flex-column">
+        <header>
+          <h1 class="section-header vw-2-margin">About</h1>
+        </header>
+        <p class="notes">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Corrupti in voluptates alias explicabo similique ab illum quas. Rerum minus assumenda adipisci possimus hic tenetur voluptatibus. Vitae laboriosam eveniet rerum quisquam.</p>
+        <p class="my-button"><span> Get my CV <i class="fa fa-download" aria-hidden="true"></i></span> </p>
       </div>
-      <div class="about-info bg-yellow">
-        <p class="about-title">Languages</p>
-        <ul>
-          <li>JavaScript (ES5/ES6)</li>
-            <li>Ruby</li>
-            <li>HTML</li>
-            <li>CSS</li>
-        </ul>
+      <div class="about-info-wrapper display-flex-column">
+        <div class="about-info bg-blue">
+          <p class="about-title">Frameworks</p>
+          <ul>
+            <li>Bootstrap</li>
+              <li>Ruby on rails</li>
+              <li>RSpec</li>
+              <li>CSS</li>
+          </ul>
+        </div>
+        <div class="about-info bg-red">
+          <p class="about-title">Frameworks</p>
+          <ul>
+            <li>Bootstrap</li>
+              <li>Ruby on rails</li>
+              <li>RSpec</li>
+              <li>CSS</li>
+          </ul>
+        </div>
+        <div class="about-info bg-yellow">
+          <div class="about-info-content">
+            <p class="about-title">Frameworks</p>
+            <ul>
+              <li>Bootstrap</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>CSS</li>
+            </ul>
+          </div>
+        </div>
+        <div class="about-info bg-empty">
+          
+        </div>
       </div>
     </section> 
-    <section class="section-title contact">
-      <header>
-        <h1 class="section-header" id="my-name">Get started</h1>
+    <section class="section-title contact display-flex-column">
+      <header class="display-flex-column">
+        <h1 class="section-header">Get started</h1>
       </header>
       <p class="notes">Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work.</p>
```

Comment:
- [ ] kindly consider removing the horizontal scroll that this website has and this will help you stick to the provide
- [ ] your page 👇 you can see its not there
![shu](https://user-images.githubusercontent.com/60146030/221301023-12c4fbb4-22d9-454a-a173-a49be50b045b.PNG)
- Figma page 👇 
![umu](https://user-images.githubusercontent.com/60146030/221301403-ac890426-e3a9-49e8-91b3-2805203b1c99.PNG)